zeppelin-web/src/app/jobmanager/jobmanager.html

```diff
@@ -34,7 +34,16 @@ <h3 class="new_h3">
   <div class="row">
     <div class="col-md-6 text-left">
       <div class="form-inline">
-        <span class="labelBtn btn-group" style="margin-left: 0px; padding-left: 10px;">
+        <span class="labelBtn btn-group">
+          <button type="button"
+                  class="btn btn-default"
+                  style="width: 25px; height: 25px; margin-right: 0px; padding: 1px 0px 3px 3px"
+                  ng-click="onChangeSortAsc()"
+                  tooltip-placement="right" tooltip="Sort by Notebook name">
```

Comment:
@cloverhearts  Since `Notebook` is a group of several notes, so`Notebook` -> `note` is more proper in here I think.